### ISSUE TITLE
Highlight is not working

### ISSUE THREAD

**Issue by ihh** [GMOD/jbrowse#1136]
Highlight seems to be broken, or is working inconsistently. I can reproduce this problem on the jbrowse.org demo using Chrome 65.0.3325.181, macOS Sierra 10.12.6. Just click on the highlight button and try to highlight by dragging on the main view: it zooms instead.

**Comment by cmdcolin** [comment]
Just curious but are you dragging along the "location bar" instead of the main view? Because if you are dragging along the location bar even with the highlight thing selected it will just do the default zoom behavior

Screenshot for illustration

![screenshot-jbrowse org-2018 07 23-14-29-15](https://user-images.githubusercontent.com/6511937/43095836-19476c8a-8e85-11e8-8c67-527fd99fdcdf.png)

**Comment by rbuels** [comment]
Yeah it seems to be working for me, if I highlight in the main area. Latest Chrome on mac.

**Comment by ihh** [comment]
Yes, I was dragging on the location bar. Apparently I do not know how jbrowse works. Closing in shame.

**Comment by cmdcolin** [comment]
It was suggested that we should probably just make dragging on location bar perform highlight. Ill reopen this :)

CC @scottcain

**Comment by scottcain** [comment]
It's definitely the case that dragging on the location bar after clicking the highlight button should highlight.  What I'd like to add is a hook to pop up a menu instead presumably with a modifier like shift or option, which would be able to modified in the config file much like you can modify the right click menu when clicking on a feature.  At WormBase, a simple drag on the details ruler (equivalent to the location bar in JBrowse) results in a popup menu, with options:

1. Zoom in
2. Dump selection as FASTA
3. Browse this section at modENCODE
4. Browse this section at UCSC
5. BLAT this sequence
6. BLAST this sequence

As you can see, we squeeze quite a bit of functionality out of this menu, so it would be nice to do this in JBrowse too.  Amusingly, highlight is not one of these options :-)

<img width="584" alt="screen shot 2018-08-10 at 8 01 53 pm" src="https://user-images.githubusercontent.com/274751/43985807-56b07c54-9cd8-11e8-92ea-6b635d0ce914.png">
